packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -572,12 +564,15 @@ contract OPContractsManagerUpgrader is OPContractsManagerBase {
             uint256 l2ChainId = getL2ChainId(IFaultDisputeGame(address(permissionedDisputeGame)));
 
             // Start by upgrading the SystemConfig contract to have the l2ChainId.
+            /*
             upgradeToAndCall(
                 _opChainConfigs[i].proxyAdmin,
                 address(_opChainConfigs[i].systemConfigProxy),
                 impls.systemConfigImpl,
+                // PEP: How to get the correct superchain config?
```

Comment:
You can get it from the OptimismPortal or something before the upgrade goes through